dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -54,7 +53,7 @@
                         :validation-messages="{
                           required: 'Select a framework to submit your request',
                         }"
-                        outer-class="long"
+                        is-required="true"
```

Comment:
manchmal isRequired, manchmal :required="true", manchmal is-required="true", funktioniert das alles gleich?
dann würde ich das einheitlich machen.